NodeOS.svg

```diff
@@ -3,6 +3,7 @@
    xmlns="http://www.w3.org/2000/svg"
    width="56cm"
    height="60cm"
+   shape-rendering="geometricPrecision"
```

Comment:
The improvements are somewhat of the same. CrispEdges produces a "color-true" version with a bit of "damage" to the borders while geometricPrecission produces a, well, geometrically precise image. It's up to you :P